supplementary/devices/taoshan.rst

```diff
@@ -0,0 +1,75 @@
+
+Taoshan - Sony Xperia L
+===========================
+
+**//Disclaimer:** Please remove everything that is just part of this template and not actually about the device before submitting this into the official wiki, thank you! **//**
+
+The device is quite weak to run Halium. But it should be possible.
+
+Status
+------
+
+Halium
+^^^^^^
+
+There is no work done, yet. But someone planning to put hands on. 
+
+* Is someone already working on Halium for this device? If YES: link to repositories of **device specific files** (usually github repos called "android_device_manufacturer_codename") and **device kernel** (usually a repo beginning with "android\ *kernel*\ ") and name authors if possible (perhaps link to their github profile)
+* Is Halium working for this device? If YES: Name the android base version (5.1 or 7.1) and everything you know about the actual status (what's working, which rootfs, ...)
+
+Distributions
+^^^^^^^^^^^^^
+
+The following entries are just a placeholder, exactly as this sentence.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Distribution
+     - Device Specific Files
+     - Kernel
+     - What works
+     - What doesn't work
+   * - LineageOS placeholder `device page placeholder <placeholder>`_
```

Comment:
Please link appropriate Lineage trees for your device

Reply:
done